bob/main.py

```diff
@@ -239,9 +239,10 @@ def weather_command(update, context):
             current_wind = w["speed"]
             current_wind_direction = wind_direction(w['deg'])
             weather_description = replace_weather_description_with_emojis(z[0]["description"])
-            weather_string = (city_name + ", " + country + " klo " + localtime.strftime("%H:%M (") + str(timezone) + ")"
-                "\n" + str(current_temperature) + " °C (tuntuu " + str(current_feels_like) + " °C)"
-                "\n" + str(current_wind) + " m/s " + str(current_wind_direction) +
+             weather_string = (city_name + ", " + country + 
+                "\n\U0001F552 " + localtime.strftime("%H:%M (") + str(timezone) + ")" +
```

Comment:
Could these be actual emoji-characters (🕒) or do they have to be in that not human-readable form? (0001F552)

Reply:
Apparently works at least in VSCode locally - don't know how it affects the bot and sending messages.

Reply:
At least some style guides recommend like this:
https://google.github.io/styleguide/javaguide.html#s2.3.3-non-ascii-characters